packages/smoldot-provider/src/SmoldotProvider.test.ts

```diff
@@ -196,6 +320,7 @@ test('subscribe', async t => {
       }
 
       t.deepEqual(result, { dummy: "state" });
+      provider.disconnect();
```

Comment:
Shouldn't this be either await or disconnect().then(() => { resolve() })?

Reply:
It doesn't acutally matter because the tests run separately in parallel and ava cancels the pending timeout implicitly for us when the test ends.  We should wait for the provider to cancel its interval pinger to be consistent and faithful to how its intended to be used though :+1: